pkg/cmd/server/origin/run_components.go

```diff
@@ -265,9 +268,15 @@ func (c *MasterConfig) RunDeploymentController() {
 	if err != nil {
 		glog.Fatalf("Unable to initialize deployment controller: %v", err)
 	}
+
+	port, err := kubeserver.GetKubernetesMasterPort(c.Options)
```

Comment:
yeah, the service port inside a pod is showing up as `443` for me, even when the master is running on `8443`